state/migration_export.go

```diff
@@ -238,9 +238,6 @@ func (st *State) exportImpl(cfg ExportConfig, leaders map[string]string, store o
 		}
 	}
 
-	export.model.SetSLA(dbModel.SLALevel(), dbModel.SLAOwner(), string(dbModel.SLACredential()))
-	export.model.SetMeterStatus(dbModel.MeterStatus().Code.String(), dbModel.MeterStatus().Info)
```

Comment:
Have you updated the description package too?

Reply:
I haven't I'm not sure how we should tackle that one. At the very least it should be ok to just leave it, as we'll drop the metrics on the floor.